### DISCUSSION TITLE
Help with selector on tooltip placement

### DISCUSSION THREAD

**Discussion by dbriard** [AvaloniaUI/Avalonia#16282] [Q&A] [answered]
Hi, I need help to customize my tooltips depending of their placement (adding an arrow).

I am trying something like that: <Style Selector="^ToolTip[(ToolTip.Placement)=Top] /template/ in the tooltip theme but it don't work.
I suppose this is because the Placement is set on the control, not on the tooltip.

Is there a syntax to access the tooltip owning control in a selector?

Alternatively, I tried something like :
```
ToolTip.IsOpenProperty.Changed.Subscribe(x =>
{
    var sender = x.Sender as Control;
    var tip = ToolTip.GetTip(sender) as Avalonia.Controls.ToolTip;
            
```
but tip is a string, not the ToolTip control.

Thanks in advance for helping.

**Comment by maxkatz6** [comment, marked as answer]
11.1.1 will get ToolTipOpening event https://github.com/AvaloniaUI/Avalonia/pull/15493

**Reply by dbriard** [reply]
Awesome, look like a nice addition to the Api!
